Classes/Command/CrawlerCommandController.php

```diff
@@ -81,13 +81,17 @@ class CrawlerCommandController extends CommandController
     /**
      * Crawl all sites
      *
+     * This command will crawl all sites based on their primary active domain.
+     *
+     * @param string|null $method method to be used for Crawling (Nodes | RobotsTxt)
+     *
      * @throws EelException
      * @throws MissingActionNameException
      * @throws PropertyException
      * @throws NeosException
      * @throws HttpException
      */
-    public function crawlSitesCommand(): void {
+    public function crawlSitesCommand(string $method = 'Nodes'): void{
```

Comment:
Would be good to use constants here and only use lowercase.
Also some documentation in the readme about the usage is needed.